src/components/parameters/ParameterBaseSequenceList.svelte

```diff
@@ -0,0 +1,107 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import ParameterName from './ParameterName.svelte';
+  import type { FormParameter, ParameterType } from '../../types/parameter.js';
+  import type { ActionArray } from '../../utilities/useActions.js';
+  import { createEventDispatcher } from 'svelte';
+  import SearchableDropdown from '../ui/SearchableDropdown.svelte';
+  import ParameterUnits from './ParameterUnits.svelte';
+  import ParameterBaseRightAdornments from './ParameterBaseRightAdornments.svelte';
+  import { userSequences } from '../../stores/sequencing';
```

Comment:
These parameter components are intended to be as generic as possible. Using a store here shoehorns the component a little bit. Rather than making this "sequence" specific, can this be some sort of generic "list" display?